src/main/java/uk/gov/companieshouse/overseasentitiesapi/utils/Constants.java

```diff
@@ -23,6 +23,7 @@ private Constants() {
     public static final String COSTS_URI_SUFFIX = "/costs";
     public static final String TRANSACTIONS_PRIVATE_API_PREFIX = "/private/transactions/";
     public static final String TRANSACTIONS_PUBLIC_API_PREFIX = "/transactions/";
+    public static final String RESUME_JOURNEY_URI_PATTERN = "/register-an-overseas-entity/transactions/%s/overseas-entity/%s";
```

Comment:
Agreed this format with @mouhajer-ch who is doing the corresponding web task. Does it look ok to everyone?

I originally had '/resume' on the end of it but took this off as we thought it didn't really belong if the web is just doing a GET of a resource with this URL.

Reply:
I think we should have the /resume on the end, it will make it easier to identify the route in the web app router and give us a name for a controller eg, resumeController in the web app

Reply:
the api get endpoint would be of the format `"/transactions/{transaction_id}/overseas-entity/{overseas_entity_id}`. with no /resume

Reply:
Hi @markpit, the endpoint is unique on the web and it is easy for the web to identify it, the web endpoint format will be something like `/register-an-overseas-entity/:transaction_id/overseas-entity/:overseas_entity_id`, but if you think the `resume` will give some sort of value we can add it

Reply:
For the payment callback endpoint we are using `/register-an-overseas-entity/transactions/:transaction_id/overseas-entity/:overseas_entity_id/payment ` with `payment` at the end, I do not mind to use the same logic with `resume` at the end

Reply:
yes i think by adding /resume it will just make it clear to the reader exactly what the url is for

Reply:
So lets add resume then 😄

Reply:
I can @mouhajer-ch. I had this originally but you suggested 'transactions' plural which I was also then happy with as it matched the example here - https://companieshouse.atlassian.net/wiki/spaces/Arch/pages/3509026858/Transactions+Model#Example-4---Resume-Journey-URI.

I really don't mind. I think if there wasn't that example I'd also go with 'transaction'. @markpit thoughts?

Reply:
I was referring to the API endpoint, for the web for consistency we are using the same format as for the payment [here](https://github.com/companieshouse/overseas-entities-web/blob/3846ecb38c3f6965e269cabe9c3d6ba8e61a5699/src/config/index.ts#L99)

Reply:
the web has used transaction (no s) for payment so i think we should have no 's' for the resume url eg
`/register-an-overseas-entity/transaction/:transactionId/overseas-entity/:overseaEntityId/resume`

the CH apis have 's' so `/transactions/{transaction_id}/overseas-entity/{overseas_entity_id}`. for the api

Reply:
Alles klar! I will amend the URL.